deploy.sh

```diff
@@ -186,7 +186,7 @@ function retry_terraform {
     # Give some breather for TF lock file to appear
     sleep 10
 
-    while [[ -f ./.terraform.tfstate.lock.info ]]; do
+    while [[ -f ./.terraform.tfstate.lock.info || $(lsof -t "$LOG_FILE" | wc -l) -gt 0 ]]; do
```

Comment:
We are writing to LOGFILE line by line.. so do you think `lsof` will help in any way here?

Reply:
`while [[ -f ./.terraform.tfstate.lock.info]] || [[ $(lsof -t "$LOG_FILE") ]];` is enough I suppose?